pkg/router/user/user.go

```diff
@@ -151,22 +165,21 @@ func userGetCreditCard(pg *db.DB, logger *zap.SugaredLogger) echo.HandlerFunc {
 // @Tags			CreditCard
 // @Accept			json
 // @Produce		json
-// @Param			credit_card	body		json.RawMessage	true	"Credit Card"
+// @Param			credit_card	body		interface{}	true	"Credit Card"
 // @Success		200			{object}	json.RawMessage
```

Comment:
might have to write a custom struct for input/return for better understanding, though it can almost pass any kind of variable in.

Reply:
since we dont check input json .I dont think we should put a struct unused in to code just for swagger demonstration

Reply:
But binder would fail if the input is invalid and output is for frontend to know what they will get

Reply:
In this case, the binder should only fail if the input is invalid JSON data, which it ideally should not be. However, even if we write the struct for demonstration purposes and provide it to the frontend as input frontend can use own struct  the demonstration still useless anyway.

Reply:
Ideally, a credit card would be involved, correct? Why not just make it easier to read and more useful?

Reply:
I don't think we need to validate input json aside from potential XSS attacks(`html.EscapeString()` prevents this btw), I think we could just an empty object for the swagger docs and use `[]byte` internally?